docs/customer_management/create_user_registration_form.md

```diff
@@ -7,13 +7,13 @@ description: Customize the registration form for new users in your site front en
 You can create a registration form for users to your website by creating a Twig template or editing the existing registration form in YAML file.
 Follow the instructions below to create and customize templates for a registration form, and a registration confirmation page.
 
-First, make sure you [enabled user registration](permission_use_cases.md#register-users).
+First, make sure you [enabled user registration](../permissions/permission_use_cases.md#register-users).
```

Comment:
Is this change needed? Autolinks should cover it.

Reply:
So I guess PhpStorm lied to me :D I will remove them

Reply:
Yes, Storm unfortunately does not understand autolinks, but autolinks are smarter ;)